storage/innobase/sync/sync0rw.cc

```diff
@@ -293,10 +293,13 @@ rw_lock_s_lock_spin(
 
 	ut_ad(rw_lock_validate(lock));
 
+	rw_lock_stats.rw_s_spin_wait_count.inc();
```

Comment:
We can decide the semantics of spin_wait_count but as I see in MySQL too spin_wait count is representative of entering spin-cycle once. So in theory if I understood the semantics correctly it would be incremented only once for each spin-cycle invocation. Moving below lock_loop will increment it every time a loop is half-interrupted.